src/osd/PG.cc

```diff
@@ -4184,13 +4192,15 @@ void PG::chunky_scrub(ThreadPool::TPHandle &handle)
              p != pg_log.get_log().log.rend();
              ++p) {
           if (cmp(p->soid, scrubber.start, get_sort_bitwise()) >= 0 &&
-	      cmp(p->soid, scrubber.end, get_sort_bitwise()) < 0) {
+	      cmp(p->soid, scrubber.end, get_sort_bitwise()) <= 0) {
```

Comment:
@dzafman @dachary  In origial PR https://github.com/ceph/ceph/pull/11255, we do not modify the compare logic(origin is p->soid < scrubber.end ). Could you tell me why we modify this line?